partiql-parser/src/main/antlr/PartiQL.g4

```diff
@@ -555,7 +555,7 @@ mathOp00
     ;
 
 mathOp01
-    : lhs=mathOp01 op=(PLUS|MINUS) rhs=mathOp02
+    : lhs=mathOp01 op=(PLUS|MINUS|AMPERSAND) rhs=mathOp02
```

Comment:
Also should add to precedence table https://github.com/partiql/partiql-lang-kotlin/blob/d0ba118c0b8507694ec30a82b4a10b7e7220006e/partiql-parser/src/main/antlr/PartiQL.g4#L485-L501